yearn/prices/band.py

```diff
@@ -13,14 +13,38 @@
     Network.Fantom: '0x56E2898E0ceFF0D1222827759B56B28Ad812f92F'
 }
 
+supported_assets = {
+    # https://docs.fantom.foundation/tutorials/band-protocol-standard-dataset#supported-tokens
```

Comment:
good addition. if they add a registry in the future, it would be preferred to hardcoding addresses.